grpcclient/grpcclient_test.go

```diff
@@ -0,0 +1,46 @@
+package grpcclient
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+func TestConfig(t *testing.T) {
+	t.Run("custom compressors", func(t *testing.T) {
```

Comment:
```suggestion
func TestCustomCompressorsConfig(t *testing.T) {
```

Reply:
I guess it's a style issue, but to me it made sense to make the root test for the `Config` class, and then have sub tests for functionalities.

Reply:
yeah, not important at all. It just look weird to me to have top-level test function with single `t.Run` call :) Feel free to ignore.

Reply:
Hopefully there will be more sub-tests :)